sdk/search/search-documents/review/search-documents.api.md

```diff
@@ -902,6 +952,14 @@ export interface KeepTokenFilter extends BaseTokenFilter {
     odatatype: "#Microsoft.Azure.Search.KeepTokenFilter";
 }
 
+// @public
+export interface KeyAuthAzureMachineLearningVectorizerParameters extends BaseAzureMachineLearningVectorizerParameters {
+    authenticationKey: string;
+    // (undocumented)
+    authKind: "key";
```

Comment:
TODO: Maor will chat with other architects about using classes instead of discriminated unions (not blocking)